packages/pluggableWidgets/datagrid-web/src/components/__tests__/Table.spec.tsx

```diff
@@ -85,7 +119,8 @@ function mockTableProps(): TableProps<ObjectItem> {
             draggable: false,
             hidable: "no" as const,
             width: "autoFill" as const,
-            size: 1
+            size: 1,
+            alignment: "left" as const
```

Comment:
This `as const` part makes it readonly. Nice, I didn't know about this syntax yet!

Reply:
Yea, if you have an enum then you can define the text as const :)